op-vis/package.json

```diff
@@ -9,6 +9,8 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.1.3",
+    "bootstrap": "^5.2.2",
```

Comment:
We may be able to remove the bootstrap css line in 

https://github.com/justinchuby/torch-onnx-op-matrix/blob/cc705d7032f4fe7df4874ba703536a8d31773fc2/op-vis/public/index.html#L26-L31

then?

Reply:
Will remove the css line!